ci/bundle_custom_data.py

```diff
@@ -13,7 +13,12 @@
 # This list is used for our CI tests to determine whether a bundle needs to be tested with
 # the `verify_data_shape` function in `verify_bundle.py`.
 # If a bundle does not need to be tested, please add the bundle name into the list.
-exclude_verify_shape_list = ["mednist_gan", "lung_nodule_ct_detection", "pathology_nuclei_segmentation_classification"]
+exclude_verify_shape_list = [
```

Comment:
cc @Can-Zhao , I skipped this check on this bundle here. Although it can be addressed after this issue: https://github.com/Project-MONAI/MONAI/issues/6486 is solved, modify the logic in bundle side is not very easy (because of the customized schema you designed).